### ISSUE TITLE
Reddit is now redirecting to reddit.com

### ISSUE THREAD

**Issue by anon238** [libredirect/libredirect#287]
This is on version 2.

**Comment by ManeraKai** [comment]
Reset the settings. We rewrote so many things and 2.0 is incompatible with 1.7

**Comment by anon238** [comment]
Yep found out that importing doesn’t work (I don’t care though), but found out that a clean install fix everything.

**Comment by remyabel2** [comment]
Am I correct in understanding that the update means the settings are incompatible and you have to set everything up again? That seems like a rather abrupt backwards incompatible change without warning.

**Comment by ManeraKai** [comment]
There's is no option in Firefox Addons to warn users about this. I'll add a mechanism that sees the version number and resets the data if it increments 0.1.0. btw this was the main reason why we stopped releasing every week or so because the extension is breaking because of fast development.

**Comment by remyabel2** [comment]
Anything is better than no notification. I skimmed the releases changelog and website and saw nothing about it. Just that the settings were no longer working and saw this issue. Even if there's no addon mechanism, opening a page in a new tab or toast notification, literally anything, would work.

I don't think that resetting the data every minor release is a good idea for a number of reasons. First, this is obscenely annoying. Users expect stability for minor point increments. Major increments makes sense, but not if you don't have any sort of warning (at least give us time to export our settings). I saw issues about import settings not working either so it's a moot point. Even an automatic migration would be an acceptable trade.

If the addon is breaking everytime you make a new release, then there's something wrong with the development methodology. I'd expect development to happen in a separate alpha/beta branch and releases to only happen if the addon is stable. If users really want to test out the newest changes, they can do so, but releasing broken versions is tantamount to making your users beta testers.
